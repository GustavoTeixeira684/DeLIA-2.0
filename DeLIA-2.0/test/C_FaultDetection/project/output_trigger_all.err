Any local data was saved: No such file or directory
Any local data was saved: No such file or directory
Any local data was saved: No such file or directory
Any local data was saved: No such file or directory
SIGNAL_RECEIVED_15_IN_RANK_1
SIGTERM RECEIVED, CLOSING DeLIA
SIGNAL_RECEIVED_15_IN_RANK_0
SIGTERM RECEIVED, CLOSING DeLIA
SIGNAL_RECEIVED_15_IN_RANK_2
SIGTERM RECEIVED, CLOSING DeLIA
SIGNAL_RECEIVED_15_IN_RANK_3
SIGTERM RECEIVED, CLOSING DeLIA
NODE_1_DO_NOT_ANSWER
SEND_TRIGGER_TO_ALL_NODES BY ID: 0
NODE_0_DO_NOT_ANSWER
SEND_TRIGGER_TO_ALL_NODES BY ID: 1
OBSERVED_NODE_2_RECEIVES_TRIGGER
MSG_RECEIVED_2_:_-1
OBSERVED_NODE_3_RECEIVES_TRIGGER
MSG_RECEIVED_3_:_-1
OBSERVED_NODE_0_RECEIVES_TRIGGER
MSG_RECEIVED_0_:_-1
OBSERVED_NODE_1_RECEIVES_TRIGGER
MSG_RECEIVED_1_:_-1
