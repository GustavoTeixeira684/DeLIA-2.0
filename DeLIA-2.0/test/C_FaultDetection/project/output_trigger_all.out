============================================
-----	 FAULT TOLERANCE 	-----
 FT_FOLDER 				 = ./data
 CHECKPOINTING_TIME_GLOBAL 		 = -1
 CHECKPOINTING_TIME_LOCAL 		 = 2
 CHECKPOINTING_TIME_DIFF_LOCAL 		 = 1
 CHECKPOINTING_GLOBAL_ITERATION 	 = -1
 TIME_SHARING_LOCAL_DATA_NEIGHBOR 	 = -1
 CHECKPOINTING_LOCAL_RECENT 		 = 1
 WORKLOAD_SCHEDULER 			  = DS
 WORKLOAD_SCHEDULER_LEADER 		 = -1
 TRIGGER_SIGNAL 			 = 1
ID: 1 É servidor com clientes: 3 0 
 TRIGGER_HEARTBEAT_MONITORING:
	 TYPE 				 = LEADER
	 TIME_MAX_WAIT			 = 20
	 SLEEP_THREAD_TIME		 = 1
============================================
ID: 0 É servidor com clientes: 2 1 
ID: 2 É cliente do servidor 0
ID: 3 É cliente do servidor 1
ID 2 (Cliente) enviou report ao servidor 0 na porta 8080
-----	 DeLIA Global Data is not actived 	-----
ID 3 (Cliente) enviou report ao servidor 1 na porta 8081
ID 0 (Servidor) enviou report ao servidor 1 na porta 8081
ID 1 (Servidor) enviou report ao servidor 0 na porta 8080
No ID: 0 É Servidor E está recebendo report do cliente: 1com comprimento: 1 do ip: 127.0.0.1 pela porta: 8083
-----	 DeLIA Global Data is not actived 	-----
-----	 DeLIA Global Data is not actived 	-----
-----	 DeLIA Global Data is not actived 	-----
-----	 DeLIA Global Data is not actived 	-----
ID 1 (Servidor) enviou report ao servidor 0 na porta 8080
No ID: 1 É Servidor E está recebendo report do cliente: 3com comprimento: 1 do ip: 127.0.0.1 pela porta: 8083
ID 2 (Cliente) enviou report ao servidor 0 na porta 8080
ID 3 (Cliente) enviou report ao servidor 1 na porta 8081
No ID: 0 É Servidor E está recebendo report do cliente: 2com comprimento: 1 do ip: 127.0.0.1 pela porta: 8083
No ID: 1 É Servidor E está recebendo report do cliente: 0com comprimento: 1 do ip: 127.0.0.1 pela porta: 8083
ID 0 (Servidor) enviou report ao servidor 1 na porta 8081
 JV - HHANDLER - ID = 0
 JV - HHANDLER - ID = 1
 JV - HHANDLER - ID = 2
 JV - HHANDLER - ID = 3
-----	 DeLIA Global Data is not actived 	-----
No: 0identificou falha no no: 1
JV - Server ID 0 vai enviar trigger pra porta 8083
No: 1identificou falha no no: 0
JV - Server ID 1 vai enviar trigger pra porta 8083
ID 2 (Cliente) recebeu trigger de -1 na porta 8083
ID 3 (Cliente) recebeu trigger de -1 na porta 8083
JV - SIGNAL RECEIVE FALSE - JV 
ID 0 (Servidor) recebeu trigger de -1 na porta 8083
ID 1 (Servidor) recebeu trigger de -1 na porta 8083
JV - SIGNAL RECEIVE FALSE - JV 
-----	 DeLIA Global Data is not actived 	-----
ID 0 (Servidor) No ID: enviou report ao servidor 1 na porta 80810 É Servidor 
E está recebendo report do cliente: 2com comprimento: 1 do ip: 127.0.0.1 pela porta: 8083
ID 2 (Cliente) enviou report ao servidor 0 na porta 8080
ID 3 (Cliente) enviou report ao servidor 1 na porta 8081
ID 1 (Servidor) enviou report ao servidor 0 na porta 8080
No ID: 1 É Servidor E está recebendo report do cliente: 3com comprimento: 1 do ip: 127.0.0.1 pela porta: 8083
-----	 DeLIA Global Data is not actived 	-----
-----	 DeLIA Global Data is not actived 	-----
-----	 DeLIA Global Data is not actived 	-----
TEST FINALIZED OK
TEST FINALIZED OK
TEST FINALIZED OK
TEST FINALIZED OK
